src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -12,9 +12,15 @@ the MIT License. See LICENSE in the project root for license information. -->
     mc:Ignorable="d">
 
     <Page.Resources>
-        <DataTemplate x:DataType="local:EnumEntry" x:Key="EnumRadioButtonTemplate">
-            <RadioButton Content="{x:Bind EnumName, Mode=OneWay}"/>
-        </DataTemplate>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="CommonResources.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+
+            <DataTemplate x:DataType="local:EnumEntry" x:Key="EnumRadioButtonTemplate">
+                <RadioButton Content="{x:Bind EnumName, Mode=OneWay}"/>
+            </DataTemplate>
+        </ResourceDictionary>

```

Comment:
idle wonder: can this template be part of COmmonResources?

Reply:
It _can_ but `x:Bind` requires code-behind since it needs to call `InitializeComponent()`, maybe I should make a `CommonTemplates.xaml` and code-behind for templates?

Reply:
In that case, ignore this request. You'd have a separate component whose job would be to hold these bindings, and that's definitely not what you need right now :) (because it is impossible)